### ISSUE TITLE
NullReferenceException is thrown on closing the New binding class error popup

### ISSUE THREAD

**Issue by tkapustina** [SpecFlowOSS/SpecFlow.Rider#135]
STR:
1. Open a feature file with a non-existing step
2. Create step - new - provide an existing class name
3. On the error dialog, click OK or X button
![image](https://user-images.githubusercontent.com/48007045/121327738-61158680-c91c-11eb-9444-b4b1ef2d5242.png)

ACTUAL:
[DEXP-591821](https://youtrack.jetbrains.com/issue/DEXP-591821)
```
`ClassName = System.NullReferenceException
Data.ThreadLocalDebugInfo = “
signal MainProtocol.UIAutomationInteractionModel.activeDialogs.[0].acceptButton.$.action.click (8878657396711815199)
-> RdDispatcher::FlushAll
”
HResult = E_POINTER=COR_E_NULLREFERENCE=80004003
Source = JetBrains.ReSharper.Feature.Services

StackTraceString = “
at JetBrains.ReSharper.Feature.Services.Intentions.Impl.DeclarationBuilders.ClassDeclarationBuilder.CreateClass(CreateClassDeclarationContext context)
at ReSharperPlugin.SpecflowRiderPlugin.QuickFixes.CreateMissingStep.CreateSpecflowStepFromUsageAction.CreateCSharpFile(ISolution solution, String className, String path, Boolean isPartial) in D:\a\SpecFlow.Rider\SpecFlow.Rider\src\dotnet\ReSharperPlugin.SpecflowRiderPlugin\QuickFixes\CreateMissingStep\CreateSpecflowStepFromUsageAction.cs:line 193
at ReSharperPlugin.SpecflowRiderPlugin.QuickFixes.CreateMissingStep.CreateSpecflowStepFromUsageAction.<>c__DisplayClass9_0.<Execute>b__3(String className, String path, Boolean isPartial) in D:\a\SpecFlow.Rider\SpecFlow.Rider\src\dotnet\ReSharperPlugin.SpecflowRiderPlugin\QuickFixes\CreateMissingStep\CreateSpecflowStepFromUsageAction.cs:line 83
at ReSharperPlugin.SpecflowRiderPlugin.QuickFixes.CreateMissingStep.CreateStepClassDialogUtil.<>c__DisplayClass4_0.<CreateDialog>b__0() in D:\a\SpecFlow.Rider\SpecFlow.Rider\src\dotnet\ReSharperPlugin.SpecflowRiderPlugin\QuickFixes\CreateMissingStep\CreateStepClassDialogUtil.cs:line 68
at JetBrains.IDE.UI.Extensions.BeContainerExtensions.<>c__DisplayClass12_0.<WithCustomButton>b__0()
at JetBrains.Collections.Viewable.SignalBase`1.Fire(T value) in /opt/buildAgent/work/b0c387553c046bbb/rd-net/Lifetimes/Collections/Viewable/Signal.cs:line 50
```

**Comment by tkapustina** [comment]
It stopped showing this notification
[https://youtrack.jetbrains.com/issue/DEXP-592652](https://youtrack.jetbrains.com/issue/DEXP-592652)

**Comment by nemesv** [comment]
It seems this is only happening in 2021.2. Can you also test in 2021.1.x?

**Comment by tkapustina** [comment]
Yep, you're right, it works in 2021.1.3

**Comment by nemesv** [comment]
This is fixed in version 1.9.1
